closingd/Makefile

```diff
@@ -66,7 +66,10 @@ CLOSINGD_COMMON_OBJS :=				\
 	common/type_to_string.o			\
 	common/utils.o				\
 	common/version.o			\
-	common/wire_error.o
+	common/wire_error.o			\
```

Comment:
The `read_peer_msg` handles this new feature in closingd and openingd?  So it still occurs even if closingd and openingd do not include new code?

Reply:
Yes, it's all inside read_peer_msg, which is why this change was a no-brainer.